test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3501,6 +3501,21 @@ def forward(self, x):
         x = torch.randn(1, 3, 2)
         self.run_test(GatherModule(), (x,))
 
+        class MyModule(torch.nn.Module):
```

Comment:
nit: it's better to keep the same pattern with the other class name under this test? Such as GatherModule.